daml-assistant/src/DA/Daml/Assistant/Install.hs

```diff
@@ -504,73 +506,77 @@ pattern RawInstallTarget_Latest = RawInstallTarget "latest"
 
 -- | Run install command.
 install :: InstallOptions -> DamlPath -> UseCache -> Maybe ProjectPath -> Maybe DamlAssistantSdkVersion -> IO ()
-install options damlPath useCache projectPathM assistantVersion = do
-    when (unActivateInstall (iActivate options)) $
-        hPutStr stderr . unlines $
-            [ "WARNING: --activate is deprecated, use --install-assistant=yes instead."
-            , ""
-            ]
+install options damlPath useCache projectPathM assistantVersion =
+    wrapErr "Running daml install command" $ do
+        when (unActivateInstall (iActivate options)) $
+            hPutStr stderr . unlines $
+                [ "WARNING: --activate is deprecated, use --install-assistant=yes instead."
+                , ""
+                ]
```

Comment:
This is daml3, can we remove `activate` instead of warning when it's used?

Reply:
https://github.com/digital-asset/daml/pull/18464